ITR/configs.py

```diff
@@ -143,43 +142,55 @@ class TargetConfig:
 
 class TabsConfig:
     FUNDAMENTAL = "fundamental_data"
-    PROJECTED_EI = "projected_ei_in_Wh"
+    PROJECTED_EI = "projected_ei" # really "projected
     PROJECTED_PRODUCTION = "projected_production"
     PROJECTED_TARGET = "projected_target"
     HISTORIC_DATA = "historic_data"
     TEMPLATE_INPUT_DATA = 'ITR input data'
+    TEMPLATE_INPUT_DATA_V2 = 'ITR V2 input data'
+    TEMPLATE_ESG_DATA_V2 = 'ITR V2 esg data'
     TEMPLATE_TARGET_DATA = 'ITR target input data'
 
 
 class PortfolioAggregationConfig:
     COLS = ColumnsConfig
 
 
+@dataclass
+class ProjectionControls:
+    LOWER_PERCENTILE: float = 0.1
+    UPPER_PERCENTILE: float = 0.9
+
+    LOWER_DELTA: float = -0.10
+    UPPER_DELTA: float = +0.03
+
+    BASE_YEAR: int = 2019
+    TARGET_YEAR: int = 2050
+    TREND_CALC_METHOD: Callable[[pd.DataFrame], pd.DataFrame] = staticmethod(pd.DataFrame.median)
+
+
+class TemperatureScoreControls(BaseModel):
+    base_year: int
+    target_end_year: int
+    tcre: quantity('delta_degC')
+    carbon_conversion: EmissionsQuantity
+    scenario_target_temperature: quantity('delta_degC')
+
+    def __getitem__(self, item):
+        return getattr(self, item)
+
+    @property
+    def tcre_multiplier(self) -> quantity('delta_degC/(t CO2)'):
+        return self.tcre / self.carbon_conversion
+
+
 class TemperatureScoreConfig(PortfolioAggregationConfig):
     SCORE_RESULT_TYPE = 'score_result_type'
-    # Unfortunately we need to cross over to interfaces.py
-    # FIXME: BASE_YEAR and TARGET_END_YEAR should agree with ProjectionControls
+    # FIXME: Sooner or later, mutable default arguments cause problems.
     CONTROLS_CONFIG = TemperatureScoreControls(
-        base_year=2019,
-        target_end_year=2050,
-        projection_start_year=2010,
-        projection_end_year=2019,
```

Comment:
looks like here were `projection_start_year` and `projection_end_year` as arguments, but not any more.
did you remove them intentionally or accidentally?

Reply:
Hello Michael,
I see you marked this comment as resolved, but I didn't understand, why.
Could you reply to this comment, mentioning, why you think it is resolved?

Reply:
Yet again, the UI of GitHub confuses me.  I typed in a comment and hit "Resolve Conversation", which silently dropped my comment.  How insanely stupid to at least not give me a warning that text I typed was going to be dropped!

The explanation is that the projection_start_year and projection_end_year were vestiges of an experiment that I don't want to push forward--they are unused and can be dropped.  As for base_year and target_end_year, those are merely moved around.  The bigger picture is that we have resolved the fact that we are no longer trying to independently define what should be the same thing in two different files (configs.py and interfaces.py).  Now ProjectionControls defines those variables, and TemperatureScoreConfig uses those definitions.  I marked it as resolved because we really have resolved a long-standing factorization problem.